app/import_functions.go

```diff
@@ -986,7 +986,7 @@ func (a *App) importUserChannels(user *model.User, team *model.Team, teamMember
 	return nil
 }
 
-func (a *App) importReaction(data *ReactionImportData, post *model.Post, dryRun bool) *model.AppError {
+func (a *App) importReaction(data *ReactionImportData, post *model.Post) *model.AppError {
```

Comment:
Interesting, all this unneeded and unused dry runs, I was expecting to use it, but clearly are not used, and are not needed, because the dryRun is always false (because the caller function is checking that before).